commands/urlib.go

```diff
@@ -94,15 +94,21 @@ var cmdURLib = &discordgo.ApplicationCommand{
 			Name:        "list",
 			Description: "List URLs",
 		},
+		{
+			Type:        discordgo.ApplicationCommandOptionSubCommand,
+			Name:        "edit",
+			Description: "EdiEdit URLs",
```

Comment:
```suggestion
			Description: "Edit URLs",
```
Also, is this part of the editor PR? Maybe it's not supposed to be in this one.